apps/skilavottord/ws/migrations/202412013210715-add-view-municipality.js

```
@@ -0,0 +1,18 @@
+'use strict'
```

Biome (1.9.4)

[error] Redundant use strict directive.

The entire contents of JavaScript modules are automatically in strict mode, with no statement needed to initiate it.
Safe fix: Remove the redundant use strict directive.


(lint/suspicious/noRedundantUseStrict)